---
permalink: /
title: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

<h3 style="text-align: right;" markdown="1">Hi, I'm Xiaoke, a Postdoctoral Associate at MIT CSAIL with expertise in bioinformatics, machine learning, and multiomic data analysis applied to complex diseases. Strong background in neuroimmunology and pharmaceutical science with combined wet lab and computational experience. Skilled in integrating scRNA-seq, scATAC-seq, bulk RNA-seq, and proteomics to uncover disease mechanisms and identify therapeutic targets. Proficient in Python, R, Linux, and advanced analytical tools, with proven ability to lead data-driven projects, collaborate across disciplines, and translate research insights into actionable solutions.</h3>

Education
======
China Pharmaceutical University (09/2016-06/2020)
* Bachelor of Science in Pharmaceutical Science (GPA 3.6/4.0)
* Award: Chinese government scholarship; University scholarship(3 years)

University of Nebraska Medical Center (08/2021-08/2025)
* PhD in Pharmaceutical Science (GPA 3.9/4.0)
* Selected courses: Biostatistical Computing (A+), Intro to Biostatistics, Machine Learning (A+), Biostatistics (A), Fundamentals of Biomolecules(A-), Intro (A+) and Advanced (A) Immunology
* Leadership position: president of China Students and Scholars Association

PhD projects
======
Unveiling the transcriptomic landscape of brain immunity in acute SIV infection via single-cell RNA sequencing (scRNA-seq). 
------
In this project, I utilized single-cell RNA sequencing (scRNA-seq) to investigate immune cell alterations in the central nervous system (CNS) during acute SIV infection. My work involved preprocessing the sequencing data using [cellranger pipelines](https://www.10xgenomics.com/support/software/cell-ranger/latest), performing data mining, and interpreting biological results. Leveraging the power of scRNA-seq for studying disease mechanisms, I obtained several important findings. In the study of [Microglia and macrophages alterations in the CNS during acute SIV infection](https://doi.org/10.1371/journal.ppat.1012168), I identified signature genes associated with distinct cellular phenotypes and mapped them to relevant biological and pathological pathways. This analysis also revealed two myeloid cell clusters with strong associations to neurodegenerative disorders. In the study of [Lymphocyte alternations in the CNS during acute SIV infection](https://doi.org/10.21203/rs.3.rs-6537361/v1), I conducted a comprehensive transcriptomic characterization of brain lymphocytes and blood CD4+ T cells, identifying a candidate lymphocyte phenotype potentially contributing to the initiation of CNS infection.Together, these findings highlight the immunological and pathological roles of diverse myeloid and lymphoid phenotypes in the brain during acute SIV infection. They provide important insights into early disease mechanisms and may inform the development of therapeutic strategies targeting this critical stage to reduce or eliminate viral reservoirs.

Transformation of brain myeloid cell populations by SIV in rhesus macaques revealed by multiomics (snRNA-seq and snATAC-seq).
------
In this project, I employed the 10x Genomics multiomics platform (snRNA-seq and snATAC-seq) to investigate host–virus interactions in the CNS during SIV infection. My work involved processing the multiomic data with the [cellranger arc pipelines](https://www.10xgenomics.com/support/software/cell-ranger-arc/latest), performing data mining, and interpreting results. [In this study](https://doi.org/10.1038/s42003-024-07443-4), I identified a unique population of microglia-like cells in which chromatin accessibility patterns diverged from corresponding RNA expression profiles. By integrating both the SIV and rhesus macaque genomes for read mapping, I was able to detect numerous SIV DNA fragments and RNA transcripts. The presence of SIV DNA fragments enabled analysis of viral integration sites and virus–host gene regulatory interactions. Through this analysis, I identified several candidate integration sites, including a notable preference near the FSTL5 gene. Furthermore, I demonstrated that SIV can exploit host transcription factors to regulate its own gene expression programs. These findings provide novel insights into virus–host regulatory dynamics in brain myeloid cells and deepen our understanding of SIV/HIV persistence and pathogenesis in the CNS.

Transcriptomic and epigenetic assessment of the morphine effect on brain myeloid cells in cART-suppressed SIV infection.
------
In this project, I again applied the 10x Genomics multiomics platform (snRNA-seq and snATAC-seq) to investigate morphine’s effects in the context of cART-treated SIV infection. My work involved processing the multiomic data using the [cellranger arc pipelines](https://www.10xgenomics.com/support/software/cell-ranger-arc/latest), performing data mining, and interpreting biological results. [In thie study (preprint)](https://pmc.ncbi.nlm.nih.gov/articles/PMC12458567/), I found that morphine exerts a profound immunosuppressive effect on brain myeloid cells during SIV infection, as revealed through both transcriptomic and epigenetic profiling. Furthermore, by integrating scRNA-seq datasets across species (human and rhesus macaque), I demonstrated that combination antiretroviral therapy (cART) substantially restores microglial homeostasis in infected animals.